### PR TITLE
Fix observability doc issues

### PR DESCRIPTION
**Pull request by tharinduwijewardane**
Fix https://github.com/wso2-enterprise/choreo/issues/4397

### CODE REVIEW

File: observelib/observe/src/main/ballerina/Module.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ var ret = observe:finishSpan(spanId);
 
 #### Attach a tag to a span
 
-It is possible to add tags to span by using the `observe:addTagToSpan()` api by providing the span id and relevant tag key and tag value.
+It is possible to add tags to span by using the `observe:addTagToSpan()` function by providing the span id and relevant tag key and tag value.
```
[the remark this answers is not in the document]
Comment (tharinduwijewardane) [reply observelib/observe/src/main/ballerina/Module.md:51]: `span id` is correct here.

File: observelib/observe/src/main/ballerina/natives.bal [hunk 2]
```diff
@@ -80,7 +80,7 @@ public function getAllMetrics() returns Metric[] = @java:Method {
     name: "getAllMetrics"
 } external;
 
-# Retrieves the specific metric that is described by the given name and tags.
+# Retrieve the specific metric that is described by the given name and tags.
```
[the remark this answers is not in the document]
Comment (tharinduwijewardane) [reply observelib/observe/src/main/ballerina/natives.bal:83]: All the other function descriptions in the file have the plural form of the verb. So I changed this also.

File: observelib/observe/src/main/ballerina/modules/mockextension/tracing.bal [hunk 3]
```diff
@@ -16,12 +16,16 @@
...
+# Holds event data.
+#
 public type Event record {
     string name;
     int timestampMicros;
     map<string> tags;
 };
 
+# Holds span data.
+#
```
[the remark this answers is not in the document]
Comment (tharinduwijewardane) [reply observelib/observe/src/main/ballerina/modules/mockextension/tracing.bal:28]: Yes, thats how the other functions descriptions were. I followed the same pattern.

File: observelib/observe/src/main/ballerina/natives.bal [hunk 4]
```diff
@@ -80,7 +80,7 @@ public function getAllMetrics() returns Metric[] = @java:Method {
...
 #
 # + name - Name of the metric to lookup.
```
[the remark this answers is not in the document]
Comment (tharinduwijewardane) [reply observelib/observe/src/main/ballerina/natives.bal:85]: All the other parameter descriptions in the file have a dot at the end. Do you want to remove them all ? What about return statements ?

[the remark this answers is not in the document]
Comment (praneesha) [reply observelib/observe/src/main/ballerina/natives.bal:85]: Add the full stop only for the main function description and remove it in all parameter descriptions including the return statements. For the guidelines, please see https://github.com/ballerina-platform/ballerina-distribution/blob/master/doc-guidelines.md.

File: observelib/observe/src/main/ballerina/Module.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ## Module Overview
 
-This module provides apis for observing Ballerina services.
-Ballerina supports Observability out of the box. This module provides user api's to make Ballerina Observability more flexible for the user.
+This module provides an API for observing Ballerina services.
+Ballerina supports Observability out of the box. This module provides a user API to make Ballerina Observability more flexible for the user.
```
[the remark this answers is not in the document]
Comment (tharinduwijewardane) [reply observelib/observe/src/main/ballerina/Module.md]: fixed

File: observelib/observe/src/main/ballerina/Module.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
...
 
 To observe Ballerina code, the '--b7a.observability.enabled=true' property should be given when starting the service
```
[the remark this answers is not in the document]
Comment (tharinduwijewardane) [reply observelib/observe/src/main/ballerina/Module.md]: fixed

File: observelib/observe/src/main/ballerina/Module.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
...
 (i.e. `ballerina run hello_world.bal --b7a.observability.enabled=true').
```
[the remark this answers is not in the document]
Comment (tharinduwijewardane) [reply observelib/observe/src/main/ballerina/Module.md]: fixed

File: observelib/observe/src/main/ballerina/modules/mockextension/tracing.bal [hunk 8]
```diff
@@ -16,12 +16,16 @@
...
 
+# Holds event data.
+#
 public type Event record {
     string name;
     int timestampMicros;
     map<string> tags;
 };
 
+# Holds span data.
```
[the remark this answers is not in the document]
Comment (tharinduwijewardane) [reply observelib/observe/src/main/ballerina/modules/mockextension/tracing.bal]: fixed

File: observelib/observe/src/main/ballerina/natives.bal (hunk 4 above)
[the remark this answers is not in the document]
Comment (tharinduwijewardane) [reply observelib/observe/src/main/ballerina/natives.bal]: fixed

File: observelib/observe/src/main/ballerina/natives.bal [hunk 9]
```diff
@@ -80,7 +80,7 @@ public function getAllMetrics() returns Metric[] = @java:Method {
...
 # + tags - The key/value pair tags that associated with the metric that should be looked up.
```
[the remark this answers is not in the document]
Comment (tharinduwijewardane) [reply observelib/observe/src/main/ballerina/natives.bal]: fixed

**Review by praneesha — APPROVED** [review]

**Pull request merged**